generator/generator.go

```diff
@@ -433,6 +441,17 @@ func main() {
 	}
 
 	switch {
+	case *splitByModule:
+		for packageName, code := range pathCode {
+			os.Mkdir(filepath.Join(*outputDir, packageName), 0755)
```

Comment:
Should it be 644?

Reply:
directories need to be executable as well